src/config/contentProviderApiConfig.json

```diff
@@ -58,6 +58,11 @@
         "FRAMEWORK_CATEGORY_INSTANCE_CREATE_URI": "/framework/v3/category/create",
         "FRAMEWORK_CATEGORY_INSTANCE_UPDATE_URI": "/framework/v3/category/update",
 
-        "TELEMETRY": "/data/v3/telemetry"
+        "TELEMETRY": "/data/v3/telemetry",
+
```

Comment:
only use v2 for channel exhaust api

Reply:
@harishgilimi : On-demand data exhaust API also has the same problem. So I used v2. Do i need to change in v3?